package.json

```diff
@@ -41,25 +44,39 @@
     "release": "yarn build && yarn changeset publish"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.26.0",
+    "@babel/eslint-parser": "^7.18.2",
+    "@types/chai": "^4.2.18",
+    "@types/chai-as-promised": "^7.1.4",
+    "@types/mocha": "^8.2.2",
+    "@types/node": "^12.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.26.0",
```

Comment:
yarn depcheck

depcheck: depcheck --config ../../.depcheckrc

Can we extend the depcheck with multiple configs?